client/packages/programs/src/ImmunisationProgramListView/ListView.tsx

```diff
@@ -57,7 +57,6 @@ const ImmunisationProgramListComponent: FC = () => {
             ? UNDEFINED_STRING_VALUE
             : rowData.vaccineCourses?.map(n => n.name).join(', '),
       },
-      'selection',
```

Comment:
sorry small side quest, noticed this column while taking docs screenshots, shouldn't be there anymore, this is done in OG